### DISCUSSION TITLE
iOS and Mac support.

### DISCUSSION THREAD

**Discussion by ManuhinSPalath** [microsoft-search/pnp-modern-search#1657] [Q&A]
Hi @wobba /team,

Can you please answer the below queries?

1. Will pnp search support ios?
2. If Yes, which are the supporting browsers in ios?
3. Will pnp search support Mac?
4. If Yes, which are the supporting browsers in Mac?

Facing "something went wrong" error in ios device with edge browser.

Thank you.

**Comment by wobba** [comment]
v4 works just fine in Edge/iOS for me. So you need to share more details on repro. We don't focus on mobile but do best effort and rely on those needing work done to contribute.

See  https://microsoft-search.github.io/pnp-modern-search/#supported-browsers for supported browsers and should work in Safari as well.

**Reply by ManuhinSPalath** [reply]
We are using V4 and getting "something went wrong" error in ios with edge browser.

**Reply by wobba** [reply]
Then we need more information on how to repro. Which version of the web parts are you using, which parts are on the page etc.
